### DISCUSSION TITLE
SnapCamera app not recognizing snap-camera-server, need help please

### DISCUSSION THREAD

**Discussion by ghost** [ptrumpis/snap-camera-server#135] [Q&A]
Hello,

Using MacOS 14.4 on a MacBook with apple silicon, I used the MacOS setup script but ran docker compose myself after the script never finished.

I followed the instructions to download the snap-camera-server and have it up and running in docker, I believe successfully. I also patched SnapCamera and confirmed its version 1.21. I am a developer so I understood the process of getting everything setup pretty well, I also made sure to edit the /etc/hosts file to connect SnapCamera with my server. 

However, no matter what my app will not connect and I am getting the Shutdown message in the app, I even confirmed with the docker logs that nothing is showing up when I open the app, even though my server is running.

Any advice would be much appreciated on what I am doing wrong.

**Comment by Jbla0712** [comment]
Hi, I don't know about Mac OS, but after reading the forums it is very possible that the snapcamera driver is no longer compatible after Sonoma 14.1. It would be logical that the driver is no longer signed

Shutdown message seems logical to me if the driver crashes or is refused. Try testing with OS 14.1 to see if the results are different.
I'm just going through it and giving guesses. I couldn't help you more and I'm allergic to this OS 👺
PS: Look log.files in snapcam it's maybe a problem with OpenGL too

**Reply by Jbla0712** [reply]
Sorry I did not understand (I do Google trad).
Yes, you will probably need to modify the script

http://studio-app.snapchat.com:8080/ is accessible with Docker ON?
